src/Models/UserIdentityModel.php

```diff
@@ -25,6 +26,146 @@ class UserIdentityModel extends Model
     ];
     protected $useTimestamps = true;
 
+    public function getAccessTokenByToken(string $token): ?AccessToken
```

Comment:
Can we differentiate these any better? Get AccessToken by _____... String? Hash? Input? Credential?

Reply:
Changed to `getAccessTokenByRawToken()`.

Reply:
That's a little better, at list it differentiates.